src/main/resources/db/migration/V1.0.00__init.sql

```diff
@@ -0,0 +1,48 @@
+CREATE TABLE store (
+  store_id BIGINT NOT NULL AUTO_INCREMENT,
```

Comment:
@marielbysoares not sure if there's points for it, but I did add a comment to every column in every table here